gulpfile.js

```diff
@@ -50,7 +47,19 @@ gulp.task('browserify', function() {
       var filteredList = files.filter(_.bind(checkFileExtension,this,".js"));
       bundleThis(filteredList);
     }});
+}
 
+// Dependency task
+gulp.task('init_module', function(callback) {
+  exec("git submodule update --init && cd html5-common && npm install && cd ..", function(err) {
+    if (err) return callback(err);
+    callback();
+  });
+});
+
+// Build All
+gulp.task('build', ['init_module'], function () {
+  browserify_fn();
```

Comment:
Gulp knows that if callback(err) gets executed, it will abort the remaining tasks. This is the stack trace if I input gibberish command:

[10:49:23] Using gulpfile ~/repos/html5-video-plugins/gulpfile.js
[10:49:23] Starting 'init_module'...
[10:49:23] 'init_module' errored after 11 ms
[10:49:23] Error: Command failed: /bin/sh: asdasdasd: command not found